instagram/src/components/LoginPage/Login.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+class LoginPage extends React.Component {
+  constructor() {
+    super();
+    this.state = { inputText: "" };
+  }
+
+  login(event) {
+    event.preventDefault();
+    let username = this.state.inputText;
+    localStorage.setItem("username", username);
+    //location.reload();
```

Comment:
If you check out the docs on MDN for Location.reload(), they would give you an example that shows window.location.reload() which is what would work in this situation.